var/spack/repos/builtin/packages/bmi/package.py

```diff
@@ -9,12 +9,13 @@
 class Bmi(AutotoolsPackage):
     """a communications framework and network abstraction layer"""
 
-    homepage = 'https://xgitlab.cels.anl.gov/sds/bmi'
-    git = 'https://xgitlab.cels.anl.gov/sds/bmi.git'
+    homepage = 'https://github.com/radix-io/bmi/'
+    git = 'https://github.com/radix-io/bmi.git'
 
     maintainers = ['carns']
 
-    version('develop', branch='master')
+    version('develop', branch='main')
```

Comment:
Why are you specifying a `develop` version pointing to `main`?  (I realize you currently only have a `main` branch`.)

Reply:
I just pushed an update to this PR that removes that.  I had left it their for compatibility reasons, but I don't think it's really needed.